fractions/jaeger/src/main/java/org/wildfly/swarm/jaeger/JaegerFraction.java

```diff
@@ -0,0 +1,84 @@
+package org.wildfly.swarm.jaeger;
+
+import org.jboss.logging.Logger;
+import org.wildfly.swarm.spi.api.Fraction;
+import org.wildfly.swarm.spi.api.annotations.Configurable;
+import org.wildfly.swarm.spi.api.annotations.DeploymentModule;
+
+import static com.uber.jaeger.Configuration.*;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+@DeploymentModule(name = "org.wildfly.swarm.jaeger", slot = "deployment")
```

Comment:
The `*.deployment` module is a side-car non-active place you can stash classes that ultimately get mixed into a deployment.

With the above line, you're adding that classes in your `*.deployment` package to every deployment.  Which might be what you want, I dunno.

I do notice that one of those classes has `@Vetoed` on it, which may cause problems if this fraction can be used without CDI, as then there will be a `NoClassDefinitionFoundException` or such when it's attempted to be loaded as a part of the deployment.  Once again, I dunno the specifics of what's being attempted or the context in which this fraction may be used.